aws_lambda_powertools/utilities/parameters/appconfig.py

```diff
@@ -100,16 +100,26 @@ def _get(self, name: str, **sdk_options) -> str:
         name: str
             Name of the configuration
         sdk_options: dict, optional
-            Dictionary of options that will be passed to the client's get_configuration API call
+            Dictionary of options that will be passed to the client's start_configuration_session API call
```

Comment:
```suggestion
            SDK options to propagate to `start_configuration_session` API call
```

Fixing older and suboptimal wording (when this was introduced)